From: SeongJae Park <sj@kernel.org>
Date: Sat, 18 Oct 2025 13:24:12 -0700
Subject: [PATCH] ==== obsolete_target selftest ====

selftests/damon: add a test for obsolete_target sysfs file

An RFC patch series [1] that add a new DAMON sysfs file for arbitrary
targets removal is under review.  Add a selftest for the feature.  The
new test uses the feature using the python wrapper of DAMON sysfs
interface, and confirm the expected internal data structure change is
made using drgn.

So this patch series may better to be a part of the other one [1] that
introduces the obsolete_target file.  But, because no significant change
is requested on the series so far, I'm posting this as an individual
RFC.

In the next version, I may merge the two series into one, to add all
related changes at one step.

[1] https://lore.kernel.org/20251016214736.84286-1-sj@kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 .../testing/selftests/damon/hkml_cv_bogus/hkml_cv_bogus_hzc5sotw  | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 tools/testing/selftests/damon/hkml_cv_bogus/hkml_cv_bogus_hzc5sotw

diff --git a/tools/testing/selftests/damon/hkml_cv_bogus/hkml_cv_bogus_hzc5sotw b/tools/testing/selftests/damon/hkml_cv_bogus/hkml_cv_bogus_hzc5sotw
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

